pkg/urbit/daemon/main.c

```diff
@@ -389,7 +398,7 @@ _main_getopt(c3_i argc, c3_c** argv)
            && u3_Host.ops_u.url_c == 0
            && u3_Host.ops_u.git == c3n ) {
     u3_Host.ops_u.url_c =
-      "https://bootstrap.urbit.org/urbit-v" URBIT_VERSION ".pill";
+      "https://bootstrap.urbit.org/props/brass.pill";
```

Comment:
@joemfb This change probably won't pass muster. Not sure what the correct solution here is though.

I have deployed the version of `/app/autoprop` from this PR to a ~zod moon, running next to a little script that uploads the .pill and .jam files it generates to bootstrap.urbit.org, using static per-prop filenames.

We could have the autoprop build pills with the old versioned naming scheme, but then need to be very good about remembering to update the autoprop config whenever necessary. The point of this side of this work is to have automated nearly-latest pills... Arguably if you find yourself trying to use a pill you can't run, you shouldn't be using this old a binary anyway?